### ISSUE TITLE
[Bug] Radar chart drawing error, it should not be two overlapping charts.

### ISSUE THREAD

**Issue by MimiZhang11** [apache/echarts#19281]
### Version

V5.0.0 - V5.4.3

### Link to Minimal Reproduction

https://echarts.apache.org/examples/zh/editor.html?c=radar&version=5.4.3

### Steps to Reproduce

1.Open the URL above to reproduce it.

### Current Behavior

From the example in the URL, I only configured one set of data, but it appears that there are two graphs that are overlaid together.

### Expected Behavior

A set of data should be plotted reasonably without overlapping parts.

### Environment

```markdown
- OS:
- Browser: Chromium version	118.0.5993.118
- Framework:
```

**Comment by helgasoft** [comment]
agree that _radar_ chart is a little confusing. Consider _radar.indicator_ as the circular axis and _series.data_ - the two polylines drawn.
See also #19182

**Comment by Ovilia** [comment]
@MimiZhang11 What do you expect to get? Two radar charts?

**Comment by MimiZhang11** [comment]
> @MimiZhang11 What do you expect to get? Two radar charts?
No, I think there should only be one graph generated from a set of data. Like this:
![44444](https://github.com/apache/echarts/assets/80016045/0cb1ab72-baf0-42cf-8b8e-40deda074cb6)

**Comment by MimiZhang11** [comment]
> @MimiZhang11 What do you expect to get? Two radar charts?

**Comment by helgasoft** [comment]
duplicate of #16925, #19236 ?

**Comment by MimiZhang11** [comment]
> duplicate of #16925, #19236 ?

Yes, it's the same error.

**Comment by MimiZhang11** [comment]
@Ovilia I also noticed a strange thing. When I opened my URL in the virtual machine, the image turned out to be normal and there were no errors, but there were errors locally.
